jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/insights/QueryEditor.java

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudwatch.logs.insights;
+
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.ui.SimpleToolWindowPanel;
+import org.jdesktop.swingx.JXDatePicker;
+import javax.swing.ButtonGroup;
+import javax.swing.JButton;
+import javax.swing.JComboBox;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JRadioButton;
+import javax.swing.JTextArea;
+import javax.swing.JTextField;
+
+public class QueryEditor {
+    public JRadioButton absoluteTimeRadioButton;
+    public JRadioButton relativeTimeRadioButton;
+    public JRadioButton searchTerm;
+    public JTextField querySearchTerm;
+    public JRadioButton queryLogGroupsRadioButton;
+    private JButton saveQueryButton;
+    private JButton retrieveSavedQueriesButton;
+    private SimpleToolWindowPanel tablePanel;
+    public JTextArea queryBox;
+    private JLabel LogGroupLabel;
+    public JXDatePicker qStartDate;
+    public JXDatePicker qEndDate;
+    public JPanel qpanel;
+    public JComboBox RelativeTimeUnit;
+    public JTextField RelativeTimeNumber;
+    private final Project project;
+    public ButtonGroup TimeRange;
+    AddRemoveLogGroupTable a;
+
+    QueryEditor(Project project){
+        this.project=project;
```

Comment:
space between equals
```suggestion
        this.project = project;
```